docs/SPEC.md

```diff
@@ -1113,15 +1137,51 @@ These common values are referred to as the group key value, and can be represent
 
 A tables schema consists of its group key, and its column's labels and types.
 
-[IMPL#463](https://github.com/influxdata/flux/issues/463) Specify the primitive types that make up stream and table types
+A table is represented using the following type:
+
+    type table = {
+        columns :: []column,
+    }
+
+
+A schema type is also defined that is useful for inspecing a table's schema ignoring the actual data.
+
+    type schema = { 
+        // List of columns on the table
+        columns :: []columnMeta, 
+    }
 
-### Stream of tables
 
-A stream represents a potentially unbounded set of tables.
-A stream is grouped into individual tables using the group key.
-Within a stream each table's group key value is unique.
+### Channel of tables
 
-[IMPL#463](https://github.com/influxdata/flux/issues/463) Specify the primitive types that make up stream and table types
+A channel represents a potentially unbounded set of tables.
+A channel is grouped into individual tables using the group key.
+Within a channel each table's group key value is unique.
+
+    type channel = {
+        tables  :: [...]table,
+    }
+
+### Stream
```

Comment:
can we riff on the `channel` idea?  would `band`  be too strange?

Reply:
Yeah I think we need better names. I just picked the first thing that came to mind. Open to ideas. Are you thinking a stream is a `band` or a channel is a `band`?